exalearn-api/ExaLearn.Dal/ModelBuilderExtensions.cs

```diff
@@ -0,0 +1,170 @@
+                Text = "Goat"
+            };
+            /*var grammarAnswers = new List<Answer>
+            {
+                new Answer
+                {
+                    Id = 1,
+                    QuestionId = 1,
+                    IsCorrect = true,
+                    Text = "London"
+                },
+                new Answer
+                {
+                    Id = 2,
+                    QuestionId = 1,
+                    IsCorrect = false,
+                    Text = "Paris"
+                },
+                new Answer
+                {
+                    Id = 3,
+                    QuestionId = 1,
+                    IsCorrect = false,
+                    Text = "Moscow"
+                },
+                new Answer
+                {
+                    Id = 4,
+                    QuestionId = 1,
+                    IsCorrect = false,
+                    Text = "Minsk"
+                }
+            };*/
+
+            /*var listeningAnswers = new List<Answer>
+            {
+                new Answer
+                {
+                    Id = 5,
+                    QuestionId = 2,
+                    IsCorrect = true,
+                    Text = "Cat"
+                },
+                new Answer
+                {
+                    Id = 6,
+                    QuestionId = 2,
+                    IsCorrect = false,
+                    Text = "Dog"
+                },
+                new Answer
+                {
+                    Id = 7,
+                    QuestionId = 2,
+                    IsCorrect = false,
+                    Text = "Cow"
+                },
+                new Answer
+                {
+                    Id = 8,
+                    QuestionId = 2,
+                    IsCorrect = false,
+                    Text = "Goat"
+                }
+            };*/
+
+            var grammarQuestion = new Question
+            {
+                Id = 1,
+                AudioFile = null,
+                AudioId = null,
+                Type = QuestionType.Grammar,
+                Text = "Capital of the England",
+                Description = "Grammar Description",
+                Level = LevelType.Beginner,
+                LevelType = LevelType.Beginner,
+                Score = 2, //change according to requirements
+            };
+
+            var listeningQuestion = new Question
+            {
+                Id = 2,
+                AudioFile = null,//some audio file
+                AudioId = null,
+                Type = QuestionType.Audition,
+                Text = "The most common animal",
+                Description = "Audition Description",
+                Level = LevelType.Beginner,
+                LevelType = LevelType.Beginner,
+                Score = 2, //change according to requirements
+            };
+
+            builder.Entity<Answer>().HasData(grammarAnswer1, grammarAnswer2, grammarAnswer3, grammarAnswer4,
```

Comment:
why are answers added then questions, how you can add answers, when questions don't exists

Reply:
> why are answers added then questions, how you can add answers, when questions don't exists

fixed

Reply:
where is fixed?